App/Components/ProgressiveImage.tsx

```diff
@@ -2,9 +2,10 @@ import React, { Fragment } from 'react'
 import { View } from 'react-native'
 import { ImageStyle, Platform } from 'react-native'
 import TextileImage from '../../TextileImage'
+import { PhotoId } from '../Models/TextileTypes'
 
 export interface IProgressiveImageProps {
-  imageId: string,
+  imageId: PhotoId,
```

Comment:
Decided that UI components that really require a certain id type should just be typed strongly.

Reply:
In contrast, if we create some general purpose/reusable UI component, it's props should just be normal types like `string` or `number`, for example.